tests/chkstk-test.cpp

```diff
@@ -1,10 +1,10 @@
 #include <gtest/gtest.h>
 
-extern "C" void chkstk_test();
+extern "C" int check_big_stack_allocation();
 
 TEST(Aarch64MinGW, CHKSTKTest) 

```

Comment:
Test name should be updated as well.

Reply:
This PR is getting large. Will follow-up in next PR to avoid merging issues.